lib/bats-core/test_functions.bash

```diff
@@ -29,7 +29,37 @@ load() {
   source "${file}"
 }
 
+#########
+#  die  #  Abort with helpful and visible message
+#########
+die() {
```

Comment:
I'd be cautious with adding such short function names without prefix in this context. That might clash with user's code.

Reply:
Point taken. Renamed to `_bats_die()` for now. I'm uncertain about the underscore, because in our project use the last few years we've found it very convenient to have a test-accessible `die MESSAGE`.

Reply:
I'd like to maintain a minimal interface on bats, because each "public" function needs to maintain backwards compatibility. The `die` function can easily be added, if needed.